client/src/Components/Student/Student.js

```diff
@@ -1,19 +1,31 @@
-import React from 'react'
+import React from 'react';
+import { GET_USER } from '../../GraphQL/index';
+import { useQuery } from '@apollo/react-hooks';
+
+const Student = () => {
+    const paramId = window.location.pathname.split('/')[2];
```

Comment:
Instead of getting the id like this, we should pass it down as a prop. That way our component wouldn't break if we decide to change the route down the line.